shared/api/container.go

```diff
@@ -138,4 +138,8 @@ type ContainerSource struct {
 
 	// API extension: container_copy_project
 	Project string `json:"project,omitempty" yaml:"project,omitempty"`
+
+	// API extension: container_backup_override_pool
+	Backup []byte `json:"backup,omitempty" yaml:"backup,omitempty"`
+	Pool   string `json:"pool,omitempty" yaml:"pool,omitempty"`
```

Comment:
As pointed out in later commits, this isn't a viable option to fix this problem.

When dealing with raw data, the best option is to do what we've done with both files and images, use HTTP headers to encode additional metadata.